budgetportal/models.py

```diff
@@ -172,6 +172,9 @@ def get_function_budgets(self):
             model_url,
             model_result.elapsed.microseconds / 1000
         )
+        if model_result.status_code == 404:
```

Comment:
Perhaps this should actually be allowed to result in exception and be considered a bug